openshift/e2e-tests-openshift.sh

```diff
@@ -194,7 +195,7 @@ EOF
 }
 
 function deploy_serverless_operator(){
-  oc apply -f openshift/serverless/operator-install.yaml
+  oc apply -f openshift/serverless/serverless-operator-1.2.0.yaml -n openshift-operators
```

Comment:
Is there a way to refer to the yaml file directly from its release page instead of copying it into this repo ?

Reply:
Can we please parameterize the version number into a global variable